typescript/phoenix-sdk/examples/placeLimitOrder.ts

```diff
@@ -0,0 +1,120 @@
+import { Connection, PublicKey, Transaction, Keypair } from "@solana/web3.js";
+import { getLimitOrderNewMakerIxs } from "../src/utils/market";
+import {
+  getSeatManagerAddress,
+  deserializeSeatManagerData,
+  getClaimSeatIx,
+  getEvictSeatIx,
+} from "../src/utils/seatManager";
+import * as assert from "assert";
+import base58 from "bs58";
+
+import * as Phoenix from "../src";
+
+export async function placeLimitOrderNoLots() {
+  let endpoint = "https://api.devnet.solana.com";
+  const connection = new Connection(endpoint);
+
+  const traderKeypair = new Keypair();
+  console.log("Trader pubkey: ", traderKeypair.publicKey.toBase58());
+  const phoenix = await Phoenix.Client.create(
+    connection,
+    endpoint,
+    traderKeypair.publicKey
+  );
+  let _airdropSig = await phoenix.connection.requestAirdrop(
+    traderKeypair.publicKey,
+    1_000_000_000
+  );
+
+  let marketAddress = new PublicKey(
+    "CS2H8nbAVVEUHWPF5extCSymqheQdkd4d7thik6eet9N"
+  );
+
+  let limitOrderPacket = Phoenix.getLimitOrderPacket(Phoenix.Side.Bid, 200, 1);
+  let marketData = phoenix.markets.get(marketAddress.toBase58())?.data;
+
+  if (marketData === undefined) {
+    throw Error("Market not found");
+  }
+
+  let limitOrderIxsNewTrader = await getLimitOrderNewMakerIxs(
+    connection,
+    marketAddress,
+    marketData,
+    traderKeypair.publicKey,
+    limitOrderPacket
+  );
+
+  let tx = new Transaction().add(...limitOrderIxsNewTrader);
```

Comment:
do we need to destructure here? why can't we just pass in the array of instructions?

Reply:
I think so, I get a type error that states I need additional info (a programId) if I don't destructure